packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js

```diff
@@ -7,4 +8,14 @@ export default class PciStoragesColdArchiveService {
     this.$q = $q;
     this.OvhApiCloudProjectUser = OvhApiCloudProjectUser;
   }
+
+  /**
+   * The implementation of this function will be done in
+   * the scope of the task MANAGER-9179 which list the Cold
+   * archives associated to the user account.
+   */
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
Do not forget to remove this once function implementation is done

Reply:
Sure